internal/controllers/configuration/zz_generated_controllers.go

```diff
@@ -953,6 +958,20 @@ func (r *KongV1KongPluginReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if err != nil {
 		return err
 	}
+
+	if r.StatusQueue != nil {
+		if err := c.Watch(
+			&source.Channel{Source: r.StatusQueue.Subscribe(schema.GroupVersionKind{
+				Group:   "configuration.konghq.com",
+				Version: "v1",
+				Kind:    "KongPlugin",
+			})},
+			&handler.EnqueueRequestForObject{},
+		); err != nil {
```

Comment:
KongPlugin controller has to subscribe to in-memory events that are going to be coming from the parser.